docs: Clarify creating worker join token (#1428)
canonical/k8s-snap: docs/canonicalk8s/snap/howto/install/custom-worker.md

- A working Kubernetes cluster deployed with the `k8s` snap

## Command line arguments

To discover the configuration options available as command line arguments,
on the control node run:

```
sudo k8s join-cluster --help
```

In this example, the name of the new worker node joining the cluster is
specified through command line arguments.

> When creating a worker join token on the control plane node, if we do not
> specify the node name, the worker node will appear in the cluster with

Reviewer: ```suggestion
If we do not specify the node name upon creating a worker join token on the control plane node the worker node will appear in the cluster with
```

> the default hostname. Specify the join token is for a `--worker` node. Also
> include the name of the worker node, for example `custom-worker`.

Reviewer: ```suggestion
the default hostname. In this example, we
include the name of the worker node: `custom-worker`. To generate the join token for a worker add the `--worker` option.
```

## Configuration file

More configuration options are available when a configuration file is specified.
Please consult the [reference page] for all of the available configuration
options and their defaults.

In this example, the configuration file provided at cluster join will set the
proxy mode of the worker machine to `ipvs`.

A join token must be generated on the control plane node of the cluster.

> Specify the join token is for a `--worker` node. We will not specify the

Reviewer: ```suggestion
To generate the join token for a worker the `--worker` option is added. We will not specify the
```
